target/linux/ramips/dts/mt7621_netgear_sercomm_bzv.dtsi

```diff
@@ -211,7 +211,6 @@
 
 	gpio_expander: i2c0gpio-expander@20{
 		#gpio-cells = <2>;
-		#interrupt-cells = <2>;
```

Comment:
I think we need to add `interrupt-controller;` instead of removing it.

ref:
https://github.com/torvalds/linux/blob/305230142ae0637213bf6e04f6d9f10bbcb74af8/Documentation/devicetree/bindings/pinctrl/semtech%2Csx1501q.yaml#L35

Reply:
Done.